convert_test.bsmap

```diff
@@ -186017,6 +186017,11 @@
 				"x": 19,
 				"y": -15
 			},
+			{
```

Comment:
Do not edit this file directly.

Instead, add a definition to the [map converter's rules](https://github.com/Bluespess/tgstation-remake/blob/master/tools/map-converter/rules.js#L111) and run the map converter on `tgstation.dme` by dragging and dropping `tgstation.dme` onto `convert.bat`